index.js

```diff
@@ -199,18 +198,40 @@ FCPClient.prototype.postCodeVersion = function (codeBuffer, notes, version, late
   console.log("postCodeVersion:", version, "bytes:", codeBuffer.length);
   if (!version || !semver.valid(version)) {
     console.log("Invalid semver version: ".red, version.toString().yellow);
-    callback(false);
+    return callback(false);
   }
 
   latest = latest.toString();
+
+  var invalid = "false";
+  if (latest === "invalid") {
+    invalid = "true";
+    latest = "false";
+  }
+
   if (latest != "true" && latest != "false") {
-    latest = true;
+    latest = "true";
```

Comment:
is there a reason this doesn't default to "false" here?  We should only explicitly be setting this flag if we can

Reply:
when pushing to dev,stg,qa this flag isn't asked for. If it defaults to false, then in those envs all pushed versions will not be latest, and so stg cxsuite will not have the code as latest which isn't ideal. The only time it's actually asked is for prod